### PR TITLE
[DONT MERGE] This is to test omnivore on torchvision video reference script

### PR DESCRIPTION
**Pull request by YosuaMichael**
We patch the torchvision video reference script to return video level accuracy

### CODE REVIEW

File: references/video_classification/train.py [hunk 1]
```diff
@@ -47,17 +51,34 @@ def train_one_epoch(model, criterion, optimizer, lr_scheduler, data_loader, devi
...
+        for video, video_idx, clip_idx, target in metric_logger.log_every(data_loader, 100, header):
             video = video.to(device, non_blocking=True)
             target = target.to(device, non_blocking=True)
-            output = model(video)
+            output = model(video, "video")
             loss = criterion(output, target)
+            
+           
+            # Iterate over batch to correctly aggregate according to video_idx
+            soft_output = torch.nn.Softmax(dim=1)(output)
```
Comment (YosuaMichael) [inline references/video_classification/train.py:74]: We aggregate the softmax!
